releasenotes/notes/host-metadata-invent-account-id-60e7f9f7abf68090.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The AWS account ID is now included in the host metadata inventory.
```

Comment:
The inventory product is not public I think. I'm not sure we need a changelog (to be double check).